tests/ui/features/@upload/sbom-upload.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Upload SBOM
+
+    Background:
+        Given User is authenticated
+
+    Scenario Outline: Verify Upload SBOM page content
+       When User visits Upload page
+       Then SBOM upload tab is selected
+       And Drag and drop instructions are visible
+       And Upload button is present
+       And Accepted file types are described
+
+    Scenario Outline: Upload single SBOM
+        Given User visits Upload page
+        When User uploads "single SBOM"
+        Then Total uploaded count is shown for "single SBOM"
+        And Results of uploading "single SBOM" are visible
+
```

Comment:
For your consideration - should we also check if the SBOM is present in the DB? And should we check for both the file and the DB record, as they are two separate things? I think the former would be much harder to check for in a deployed instance, though, so maybe it's out of scope for now.

Reply:
Yeah, we should have the steps for that implemented, so that should be easy to do.